soh/soh/stubs.c

```diff
@@ -12,12 +12,12 @@ OSViMode osViModeNtscLan1;
 OSViMode osViModeMpalLan1;
 OSViMode osViModeFpalLan1;
 OSViMode osViModePalLan1;
-AudioContext gAudioContext;
-unk_D_8016E750 D_8016E750[4];
+// AudioContext gAudioContext;
```

Comment:
Context for removal?

Reply:
These are already defined somewhere else and cause linkage conflicts. I don't even know why MSVC's linker accepts this.